jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash

```diff
@@ -12,9 +12,8 @@ export HOMEBREW_MAKE_JOBS=${MAKE_JOBS}
 # Get project name as first argument to this script
 PROJECT=$1 # project will have the major version included (ex gazebo2)
 PROJECT_ARGS=${2}
-
-PROJECT_PATH=${PROJECT}
-
+# PROJECT_PATH can be passed as env variable or assume that is the same than project name
+PROJECT_PATH=${:-PROJECT}
```

Comment:
~~~
./scripts/jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash: line 16: ${:-PROJECT}: bad substitution
~~~

Reply:
arg, sorry for that [2966b86](https://github.com/gazebo-tooling/release-tools/pull/1068/commits/2966b862cd622cd877b83510e1f5241ce56fc855)